packages/blade/src/components/Toast/Toast.native.tsx

```diff
@@ -0,0 +1,13 @@
+import type { ToastProps } from './types';
+import { throwBladeError } from '~utils/logger';
+
+const Toast = (_props: ToastProps): React.ReactElement => {
+  throwBladeError({
+    message: 'Toast is not yet implemented for native',
+    moduleName: 'Toast',
+  });
+
+  return <></>;
```

Comment:
Add a basic Text here saying that toast is not available? So its easily visible to people?

Reply:
We are throwing an error on ReactNative. So it will give users a fullscreen error anyways.